src/test/scala/com/stackstate/stackpack/splunk/SplunkProvisioningTest.scala

```diff
@@ -27,7 +28,7 @@ class SplunkProvisioningTest extends WordSpecLike with Matchers {
       stackPackPackage.install(provisioningContext, config.asJava).run()
 
       val (_, templateFile, args) = verifyTemplateImportedWithNamespace(provisioningContext.stackPack.namespace)
-      args shouldBe empty
+      args should not be empty
```

Comment:
I would not expect args on the shared templates.

Reply:
fixed.